app-text/scdoc/scdoc-1.3.1.ebuild

```diff
@@ -18,7 +18,7 @@ src_prepare() {
 
 	sed -e 's/-Werror//' \
 		-e 's/CFLAGS=/CFLAGS+=/' \
-		-e '/^PREFIX/s@=.*$@=/usr@' \
+		-e 's/PREFIX=/PREFIX?=/' \
```

Comment:
Why do you need to sed it at all? CLI override should work either way.

Reply:
scdoc-1.3.1/Makefile has `PREFIX=/usr/local' so unless I'm wrong that unconditionally sets it to
`/usr/local` and cannot be overriden.

Reply:
You're wrong.

Reply:
Wouldn't be the first time. Cool, I'll nix that sedex and give it another go once I'm entirely free
(away from my workstation right now) and fix it up.